src/main.jsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App.jsx';
+import './index.scss';
 import { Provider } from 'react-redux';
 import store from './redux/store';
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+const root = ReactDOM.createRoot(document.getElementById('root'));
+
+root.render(
   <React.StrictMode>
     <Provider store={store}>
-    <App />
-  </Provider>
-  </React.StrictMode>,
-)
+      <App />
```

Comment:
- [ ] I will make this comment here because the file was not part of the changed file in this PR, according to the project requirement, you should have a filter stateless function on your home page that users can use to filter your Mario, in other words, add a search bar where users can search Mario game by name or by edition